### PR TITLE
add feature: add option to set segment length scaling factor in compute

### PR DESCRIPTION
**Pull request by armantekinalp**
solves #365 . It is a very short PR. Can you please check quickly?

### CODE REVIEW

**Review by Ali-7800 — APPROVED** [review]
LGTM!

File: elastica/rod/knot_theory.py [hunk 1]
```diff
@@ -130,13 +132,16 @@ def compute_link(
         type_of_additional_segment : str
             Determines the method to compute new segments (elements) added to the rod.
             Valid inputs are "next_tangent", "end_to_end", "net_tangent", otherwise program uses the center line.
+        alpha : float
+            Empirical factor to scale the segment length, where segment length is the rest length of the rod.
```
Comment (skim0119) [inline elastica/rod/knot_theory.py:136]: Maybe add a little description on how to use?

**Review by skim0119 — CHANGES_REQUESTED** [review]

File: elastica/rod/knot_theory.py (hunk 1 above)
Comment (armantekinalp) [reply elastica/rod/knot_theory.py:136]: I added. @skim0119 can you check and approve.

**Review by skim0119 — APPROVED** [review]
LGTM!

**Pull request merged**